scripts/Docker/readme.md

```diff
@@ -0,0 +1,79 @@
+# Librealsense Docker
+
+This tutorial aim is to provide instructions for installation and use of Librealsense Docker. 
+Current version of the docker includes the following capabilities:
+- use of librealsense devices
+- use of librealsense API
+- installation of the basic examples for use of librealsense
+
+It does not include (may be enabled later on):
+- graphic examples
+- use of IMU devices
+
+## Pre-Work: Docker Installation
```

Comment:
Specify that the prebuild docker is avail. for X86 only and provide direct link to the repo.

Reply:
Done